### PR TITLE
Handlers

### PR DESCRIPTION
**Pull request by dnozdrin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,4 +11,6 @@ vendor/
 .idea/
 
 # Project specific paths
-/.env
\ No newline at end of file
+/.env
+/log/
+/bin/detask
```
Comment (eugeneradionov) [inline .gitignore:16]: as a tip: you can add `/bin/*` to ignore all built binaries, so you can reuse this gitignore file in future projects

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.PHONY: build dependency unit-test integration-test swagger-start swagger-stop
+
+.EXPORT_ALL_VARIABLES:
+
+DB_HOST=localhost
+DB_PORT=20432
+DB_NAME=postgres
+DB_USER=postgres
+DB_PASS=testing
```
Comment (eugeneradionov) [inline Makefile:9]: do not save application configuration data in the Makefile, use config file instead
do not save sensitive data in the repository

Comment (dnozdrin) [reply Makefile:9]: are those statements also true for local-only testing scripts?

Comment (dnozdrin) [reply Makefile:9]: > do not save application configuration data in the Makefile, use config file instead
> do not save sensitive data in the repository

are those statements also true for local-only testing scripts?

Comment (eugeneradionov) [reply Makefile:9]: yep, you can create config sample with non-sensetive predefined data and use it instead

Comment (dnozdrin) [reply Makefile]: moved application configuration to the .env for local tests - https://github.com/dnozdrin/detask/pull/6/commits/863c952c5ad890895f9af6211cc689034e06c1d8

File: .gitignore (hunk 1 above)
Comment (dnozdrin) [reply .gitignore]: Done

**Pull request merged**